File: php/src/PayCore/Stablecoin.php
<?php

declare(strict_types=1);

namespace PayKit\PayCore;

/**
 * Stablecoin symbol used as a settlement-asset preference. Backing
 * values are the canonical uppercase tickers that travel on the wire.
 */
enum Stablecoin: string
{
    case Usdc  = 'USDC';
    case Usdt  = 'USDT';
    case Usdg  = 'USDG';
    case Pyusd = 'PYUSD';
    case Cash  = 'CASH';
}

File: python/src/solana_pay_kit/_paycore/stablecoin.py
"""Stablecoin symbols MPP charge and x402 exact can settle in."""

from __future__ import annotations

from enum import StrEnum

__all__ = ["Stablecoin"]


class Stablecoin(StrEnum):
    """SPL stablecoin symbol used as a settlement asset."""

    USDC = "USDC"
    USDT = "USDT"
    USDG = "USDG"
    PYUSD = "PYUSD"
    CASH = "CASH"
